web/styles/compose.css

```diff
@@ -1469,11 +1469,11 @@ textarea.new_message_textarea {
         color: var(--color-text-default);
 
         .stream-privacy-type-icon {
-            font-size: 13px;
-            width: 13px;
-            height: 13px;
+            font-size: 0.9285em; /* 13px at 14px em */
```

Comment:
With the font-size set to 13px here, which should round up to 0.9286, the `top` value below should be 2/13, `0.1538`, and the confusing conversion comment simplified.

@evykassirer FYI, if you can put up and check a follow-up for these matters, since the PR has been merged here.

Reply:
#33528